### ISSUE TITLE
Prepare for CRAN submission

### ISSUE THREAD

**Issue by trashbirdecology** [TrashBirdEcology/bbsAssistant#44]
Prepare for release: 
 
* [x] Check that description is informative 
    - [ ] Add relevant copyrights to libraries used (e.g. dplyr, magrittr, etc [see imports and depends in DESCRIPTION.md]) (this was a task added by JLB)
    - [x] Add relevant copyrights to the licensing  (this was a task added by JLB) 
* [x ] Check licensing of included files  
     - [x] make sure we incorporate the necessary copy rights or licenses from rBBS (CC0)
     - [x] make sure we incorporate the necessary copy rights or licenses for the N.Amer BBS    (CC0)                                                 
* [ ] `usethis::use_cran_comments()` 
* [x] `devtools::check()`  
     - [x] address all issues appearing in the checks (e.g. failures; no visible bindings; malformed descri. fields) (@GabsPalomo)
 * [ ] `devtools::check_win_devel()` 
* [] create package website (added by JLB ) see [here](https://pkgdown.r-lib.org/articles/pkgdown.html) for details
* [x] Polish pkgdown reference index 
* [ ] Draft blog post 
 
Submit to CRAN: 
 
* [x] `usethis::use_version('major')` 
* [x] Update `cran-comments.md` 
* [x] `devtools::submit_cran()` 
* [x] Approve email 
 
Wait for CRAN... 
 
* [ ] Accepted :tada: 
* [ ] `usethis::use_github_release()` 
* [ ] `usethis::use_dev_version()` 
* [ ] Update install instructions in README 
* [ ] Finish blog post 
* [ ] Tweet 
* [ ] Add link to blog post in pkgdown news menu

**Comment by TrashBirdEcology** [comment]
A major to do  re: DESCRIPTION + LICENSING

- [ ] Add relevant copyrights to libraries used (e.g. dplyr, magrittr, etc [see imports and depends in DESCRIPTION.md]).
- [ ] Add relevant copyrights to the licensing

**Comment by atyre2** [comment]
Have you considered getting this onboarded with rOpenSci? either before or after CRAN submission.

**Comment by trashbirdecology** [comment]
No, but that's a great idea...would be a good excuse to redo the function
naming convention to object_verb (currently is verb_objObj).

**Comment by TrashBirdEcology** [comment]
@lsw5077  or @GabsPalomo any interest in taking the lead on ticking off these boxes?
